Cartfile

```diff
@@ -6,10 +6,10 @@ github "kif-framework/KIF"          "master"
 github "ZaBlanc/RaptureXML"         "master"
 github "mozilla/readability"        "master"
 github "rs/SDWebImage"              "master"           # To get the fix for libwebp
-github "Masonry/Snap"               "Swift-2.0"
+github "Masonry/Snap"               "0.16.0"
```

Comment:
[0.17.0](https://github.com/SnapKit/SnapKit/releases/tag/0.17.0) is released 2 days ago, and the repo is renamed / transferred to [SnapKit/SnapKit](https://github.com/SnapKit/SnapKit).